styles.css

```diff
@@ -1,288 +1,287 @@
+/* stylelint-disable no-missing-end-of-source-newline */
```

Comment:
- [ ] Kindly remove this line that is meant to disable linters from checking your syntax. It is really important that you learn to avoid linting error issues by using appropriate industry-recommended standards. It makes you more suitable for working environments :pleading_face:

Reply:
Regarding this line, I added it at the beginning because without it stylint will show an error called (no-missing-end-of-source-newline) which won't be solved unless I add that quick fix suggested by vscode.
```suggestion
/* stylelint-disable no-missing-end-of-source-newline */
![Kazam_screenshot_00073](https://user-images.githubusercontent.com/86020057/227753131-89dd65a2-fed3-40d3-91e5-00ec34baeae8.png)
![Kazam_screenshot_00074](https://user-images.githubusercontent.com/86020057/227753133-128c9cb7-44b3-4dd6-853c-11faa46c5a60.png)
![Kazam_screenshot_00075](https://user-images.githubusercontent.com/86020057/227753135-850ff001-0192-48dd-be84-bfc377e7952a.png)
![Kazam_screenshot_00076](https://user-images.githubusercontent.com/86020057/227753138-68dc4f86-8c3f-4eb6-9a64-51062a97a820.png)
![Kazam_screenshot_00077](https://user-images.githubusercontent.com/86020057/227753140-223203cf-c993-4a98-83f9-44c868676892.png)

```